### PR TITLE
Fix Google Closure API endpoint to HTTPS

### PR DESCRIPTION
**Pull request by ylakim**
The HTTP endpoint of the Google Closure API doesn't seem reliable anymore. It has been (rightfully) moved to HTTPS (https://closure-compiler.appspot.com/compile).

### CODE REVIEW

File: src/Assetic/Filter/GoogleClosure/CompilerApiFilter.php [hunk 1]
```diff
@@ -89,13 +89,14 @@ public function filterDump(AssetInterface $asset)
...
-            $response = file_get_contents('http://closure-compiler.appspot.com/compile', false, $context);
+            $response = file_get_contents('https://closure-compiler.appspot.com/compile', false, $context);
             $data = json_decode($response);
         } elseif (defined('CURLOPT_POST') && !in_array('curl_init', explode(',', ini_get('disable_functions')))) {
-            $ch = curl_init('http://closure-compiler.appspot.com/compile');
+            $ch = curl_init('https://closure-compiler.appspot.com/compile');
             curl_setopt($ch, CURLOPT_POST, true);
             curl_setopt($ch, CURLOPT_HTTPHEADER, array('Content-type: application/x-www-form-urlencoded'));
             curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+            curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
```
Comment (NicolasCARPi) [inline src/Assetic/Filter/GoogleClosure/CompilerApiFilter.php:99]: Why is this line needed? The certificate should be verified!

Comment (ylakim) [reply src/Assetic/Filter/GoogleClosure/CompilerApiFilter.php:99]: The goal was to make sure the HTTPS transition wouldn't break some current installations of Assetic. But I agree it is a compromise on security.

See: https://snippets.webaware.com.au/howto/stop-turning-off-curlopt_ssl_verifypeer-and-fix-your-php-config/

**Comment by LukeTowers** [comment]
This has been implemented in the 2.0 branch of [assetic-php/assetic](https://github.com/assetic-php/assetic). Until we tag 2.0, you can pull it in by using `"assetic/framework": "dev-2.0/dev"` in your composer.json files.
